src/app/(auth)/login/page.tsx

```
@@ -1,9 +1,20 @@
-import React from 'react';
+'use client';
+import { InputWindow } from '@/app/components/InputWindow';
```

ESLint

[error] 'InputWindow' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)